nucleus/core/bootstrap/src/main/java/fish/payara/boot/runtime/BootCommands.java

```diff
@@ -59,7 +61,12 @@
  */
 public class BootCommands {
 
-    private List<BootCommand> commands;
+    private final List<BootCommand> commands;
+
+    private static final Pattern ENV_VAR_PATTERN = Pattern.compile("([^\\$]*)\\$\\{([^\\}]*)\\}([^\\$]*)");
+    private static final int MAX_SUBSTITUTION_DEPTH = 100;
+
+    private static final Logger LOG = Logger.getLogger(BootCommands.class.getName());
```

Comment:
Just to stay inkeeping with current conventions, could this be called `LOGGER` please?